blue_ai/envs/transient_goals.py

```diff
@@ -7,21 +9,22 @@
 from minigrid.core.world_object import Wall
 import numpy as np
 import imageio
-import os
 from enum import IntEnum
 
+from dataclasses import dataclass
```

Comment:
Let's leave this out unless there's a compelling reason to include it

Reply:
Oh oops that was meant to be removed